client.go

```diff
@@ -17,12 +17,14 @@
 package goovn
 
 import (
-	"sync"
-
 	"github.com/ebay/libovsdb"
+	"sync"
 )
 
-// Client ovnnb client
+var db string
```

Comment:
Sorry I didn't see where is this used. I think it should be assigned the DB name that the client is connected to, and then it can be used in each API implementation to ensure the API is called with the correct DB and raise error if it is a mismatch.